### PR TITLE
Add crontab

### PR DESCRIPTION
**Pull request by mcs07**

### CODE REVIEW

File: recipes/crontab/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    - python
+
+test:
+  imports:
+    - crontab
+
+about:
+  home: https://github.com/josiahcarlson/parse-crontab
+  license: LGPL-2.1 or LGP-3.0
+  license_family: LGPL
```
[the remark this answers is not in the document]
Comment (CJ-Wright) [reply recipes/crontab/meta.yaml]: Yes, this needs a license (as per LGPL) @conda-forge/core can we put a list of licenses/licence_file?

[the remark this answers is not in the document]
Comment (jakirkham) [reply recipes/crontab/meta.yaml]: `license` is pretty free-form. So listing things already happens here and is fine.

Adding multiple `license_file`s is not yet supported. ( https://github.com/conda/conda-build/issues/1247 )

[the remark this answers is not in the document]
Comment (mcs07) [reply recipes/crontab/meta.yaml]: Upstream request for license files in package: https://github.com/josiahcarlson/parse-crontab/issues/28

**Comment by mcs07** [comment]
A fast upstream update now means that the license file doesn't need to be included with the recipe. I've updated it accordingly.

Is this now good to merge?

**Pull request merged**